server/services/ai.py

```diff
@@ -31,6 +32,7 @@
 # ----------------------------- INTERNAL FUNCTIONS -----------------------------
 
 _INFERENCE_CLIENT = None
+_LANGUAGE_CLIENT = language_v1.LanguageServiceClient()
```

Comment:
i believe this isn't thread safe, are you seeing a lot of overhead to this call? this should go in the constructor and stay within the class.

Reply:
I think imports are thread safe so variable module initialization should be thread safe. Which constructor are you suggesting?

Reply:
Let me know if this is a thing we should fix. In that case I'll do it in a later PR.

Reply:
yes please, could you move it into the InferenceClient constructor?  since there is a way to avoid the global construction

https://stackoverflow.com/questions/35309042/python-how-to-set-global-variables-in-flask
https://stackoverflow.com/questions/32815451/are-global-variables-thread-safe-in-flask-how-do-i-share-data-between-requests